### DISCUSSION TITLE
Portainer deploy instructions

### DISCUSSION THREAD

**Discussion by ArthurMitchell42** [firefly-iii/firefly-iii#9240] [General]
Morning, I've followed the instructions to deploy using Portainer but at deploy all I get is this message:
```
Deployment error
Failed to deploy a stack: 
env file /data/compose/413/stack.env not found: 
stat /data/compose/413/stack.env: no such file or directory 

```
This is on Portainer 2.21.1

Should I just switch back to a YAML 'environment:' list?

**Comment by JC5** [comment]
Did you add the variables from the stack.env file to the configuration?

**Reply by ArthurMitchell42** [reply]
Hi, I'm working through Portainer and there seems to be an issue with its generation of the variables file. I didn't feel it was worth pursuing as a root cause investigation so I switched to just putting the variables in as a YAML list under environment: 
Not the neatest solution but it works.
